oidc-controller/api/routers/oidc.py

```diff
@@ -212,6 +212,20 @@ async def get_authorize(request: Request, db: Database = Depends(get_db)):
         "controller_host": controller_host,
         "challenge_poll_uri": ChallengePollUri,
         "wallet_deep_link": wallet_deep_link,
+        "title": (
+            ver_config.metadata.title.get("en")
```

Comment:
It might be best to use a parameter for the language, so that we don;t have to update this condition when parsing a language dynamically

Reply:
Like a parameter for the function or for the template?

Reply:
I think what you did will work, once we have multilanguage support in the UI we just need to update the code to grab that value rathe rthan the hardcoded `en` that now only appears in one spot